casper/src/test/scala/coop/rchain/casper/RholangBuildTest.scala

```diff
@@ -23,11 +23,15 @@ class RholangBuildTest extends FlatSpec with Matchers {
     import node._
 
     val code =
-      """new double, dprimes, rl(`rho:registry:lookup`), ListOpsCh in {
+      """new double, dprimes, rl(`rho:registry:lookup`), ListOpsCh, time(`rho:block:timestamp`), timeRtn, stdout(`rho:io:stdout`) in {
         |  contract double(@x, ret) = { ret!(2 * x) } |
         |  rl!(`rho:id:dputnspi15oxxnyymjrpu7rkaok3bjkiwq84z7cqcrx4ktqfpyapn4`, *ListOpsCh) |
         |  for(@(_, ListOps) <- ListOpsCh) {
         |    @ListOps!("map", [2, 3, 5, 7], *double, *dprimes)
+        |  } |
+        |  time!(*timeRtn) |
+        |  for (@timestamp <- timeRtn) {
+        |   stdout!(timestamp)
```

Comment:
This is insufficient as a test of this feature in my opinion. I would prefer to see something added to `HashSetCasperTest ` that actually checks the output is correct.